README.md

```diff
@@ -1,2 +1,64 @@
 # MAGNUM
 A Modality-Agnostic Multimodal Modular Architecture
+
+## Installation
+
+1. Clone the repository
+2. Install the requirements file with `pip install -r requirements.txt`
```

Comment:
Let's be more precise. Rather than "installing the requirements file", let's say "Install the requirements with `pip install -r requirements.txt`"